cmd/karavictl/cmd/role.go

```diff
@@ -67,6 +67,20 @@ func NewRoleCmd() *cobra.Command {
 		},
 	}
 
+	roleCmd.PersistentFlags().StringP("admin-token", "f", "", "Path to admin token file; required")
```

Comment:
Do we also need this changes to role_create(delete/update/list).go?

Reply:
No, because these are persistent flags. We can define persistent flags at the `karavictl role` level and the sub-commands will be able to use that flag. This is written this way so that all karvictl role, rolebinding, storage, tenant, and generate commands require these flags. The karavictl cluster-info and admin token commands do not.